### DISCUSSION TITLE
Cheaper alternative of the Hyperpixel?

### DISCUSSION THREAD

**Discussion by srwxr-xr-x** [FundamentalFrequency/.github#50] [Q&A]
Hey all! 
I was getting all of the parts, and the screen was the most expensive and I don't even use the touchscreen!
Is there a cheaper alternative I can use?

**Comment by stonepreston** [comment]
Theres a non touch version thats a bit cheaper, does not look as nice though. Have yet to find a screen that looks as good as the hyperpixel does.

**Reply by DominusFulguris** [reply]
I have another alternative on order. Will have to see how much work it takes to get it running once it arrives.  It says it is "plug and play"...

https://www.aliexpress.com/item/1005004473902355.html

**Reply by srwxr-xr-x** [reply]
Interesting, I might try using a HDMI Screen until I find a good alternative, That should work correct?

**Comment by briwil** [comment]
> Hey all! 
> I was getting all of the parts, and the screen was the most expensive and I don't even use the touchscreen!
> Is there a cheaper alternative I can use?

I will say that I use the touchscreen aspect to turn off the raspberry pi when I'm done with the LMN-3 (vs just unplugging it which I hear can lead to corrupted SD cards; also hopeful touch implementation could come someday as a alt to button-pushing menu-diving).
